mllib/src/main/scala/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.scala

```diff
@@ -204,4 +204,76 @@ class BinaryClassificationMetrics(
       (x(c), y(c))
     }
   }
+
+  /**
+   * Returns the calibration or reliability curve,
+   * which is an RDD of (average score in bin, fraction of positive examples in bin).
```

Comment:
This description of the RDD contents doesn't match with the `((Double, Double), (Double, Long))` type signature